### PR TITLE
[fr] Jetstream : Logout -> log out

### PR DESCRIPTION
**Pull request by SilvRO**
All translation keys containing "logout" were not functional with a fresh install. You should use "log out" instead.

### CODE REVIEW

File: json/fr.json [hunk 1]
```diff
@@ -77,10 +77,10 @@
     "Leave Team" : "Quitter l'équipe",
     "Leave" : "Quitter",
     "Login": "Connexion",
-    "Logout Other Browser Sessions" : "Déconnecter les sessions ouvertes sur d'autres navigateurs",
-    "Logout": "Déconnexion",
```
Comment (andrey-helldar) [inline json/fr.json:81]: Keys used in Laravel products cannot be deleted.

File: json/fr.json [hunk 2]
```diff
@@ -77,10 +77,10 @@
     "Leave Team" : "Quitter l'équipe",
     "Leave" : "Quitter",
     "Login": "Connexion",
-    "Logout Other Browser Sessions" : "Déconnecter les sessions ouvertes sur d'autres navigateurs",
```
Comment (andrey-helldar) [inline json/fr.json:80]: Keys used in Laravel products cannot be deleted.

File: json/fr.json [hunk 3]
```diff
@@ -77,10 +77,10 @@
...
-    "Logout": "Déconnexion",
+    "Log Out Other Browser Sessions" : "Déconnecter les sessions ouvertes sur d'autres navigateurs",
+    "Log Out": "Déconnexion",
     "Manage Account" : "Gérer le compte",
-    "Manage and logout your active sessions on other browsers and devices." : "Gérer et déconnecter vos sessions actives sur les autres navigateurs et appareils.",
```
Comment (andrey-helldar) [inline json/fr.json:83]: Keys used in Laravel products cannot be deleted.

See: https://github.com/Laravel-Lang/lang/blob/master/script/en/en.json#L86-L87

File: json/fr.json [hunk 4]
```diff
@@ -106,7 +106,7 @@
     "Please confirm access to your account by entering the authentication code provided by your authenticator application.": "Veuillez confirmer l'accès à votre compte en entrant le code d'authentification fourni par votre application d'authentification.",
     "Please confirm your password before continuing.": "Veuillez confirmer votre mot de passe avant de continuer",
     "Please copy your new API token. For your security, it won't be shown again." : "Veuillez copier votre nouveau token API. Pour votre sécurité, il ne sera pas ré-affiché.",
-    "Please enter your password to confirm you would like to logout of your other browser sessions across all of your devices." : "Veuillez entrer votre mot de passe pour confirmer que vous voulez déconnecter toutes les autres sessions navigateur sur l'ensemble de vos appareils.",
```
Comment (andrey-helldar) [inline json/fr.json:109]: Keys used in Laravel products cannot be deleted.

See: https://github.com/Laravel-Lang/lang/blob/master/script/en/en.json#L112-L113

**Review by andrey-helldar — CHANGES_REQUESTED** [review]

File: json/fr.json (hunk 4 above)
Comment (SilvRO) [reply json/fr.json:109]: You are right, but there are still some errors in the keys. Some are not present in fr.json 
Example with "logout" : 
 - https://github.com/Laravel-Lang/lang/blob/master/script/en/en.json#L79-L80
 - https://github.com/Laravel-Lang/lang/blob/master/json/fr.json#L81

The "Log Out" variant used by Jetstream is not present in fr.json

File: json/fr.json (hunk 1 above)
Comment (luisprmat) [reply json/fr.json:81]: Although 'Login' and 'Logout' were superseded in the latest versions of all Laravel products

File: json/fr.json (hunk 4 above)
Comment (andrey-helldar) [reply json/fr.json:109]: You did not understand. What I'm saying is that you have removed the option from "logout", although you need to leave both.

File: json/fr.json (hunk 1 above)
Comment (andrey-helldar) [reply json/fr.json:81]: Oh... https://github.com/laravel/jetstream/pull/677/files?file-filters%5B%5D=.php

**Comment by andrey-helldar** [comment]
@caouecs, see: https://github.com/laravel/jetstream/pull/677/files?file-filters%5B%5D=.php

**Review by andrey-helldar — APPROVED** [review]

**Comment by caouecs** [comment]
If we accept for French version, we need accept for all other languages

**Comment by andrey-helldar** [comment]
@caouecs, I prefer to keep these keys as backward compatibility until we publish version 9.0 😊

@SilvRO, undo key deletion, please.

File: json/fr.json (hunk 2 above)
Comment (andrey-helldar) [inline json/fr.json:80]: Undo key deletion.

File: json/fr.json (hunk 3 above)
Comment (andrey-helldar) [inline json/fr.json:83]: Undo key deletion.

File: json/fr.json (hunk 4 above)
Comment (andrey-helldar) [inline json/fr.json:109]: Undo key deletion.

File: json/fr.json (hunk 1 above)
Comment (andrey-helldar) [inline json/fr.json:81]: Undo key deletion.

File: json/fr.json [hunk 5]
```diff
@@ -63,7 +63,7 @@
     "Hello!": "Bonjour !",
     "hi": "salut",
     "I agree to the :terms_of_service and :privacy_policy": "Je suis d'accord avec :terms_of_service et :privacy_policy",
-    "If necessary, you may logout of all of your other browser sessions across all of your devices. Some of your recent sessions are listed below; however, this list may not be exhaustive. If you feel your account has been compromised, you should also update your password." : "Si nécessaire, vous pouvez vous déconnecter de toutes vos sessions de navigateur de tous vos appareils. Certaines de vos sessions sont listées plus bas ; pourtant, cette liste peut ne pas être exhaustive. Si vous sentez que votre compte a été compromis, vous pouvez aussi mettre à jour votre mot de passe.",
```
Comment (andrey-helldar) [inline json/fr.json:66]: Undo key deletion.

**Review by andrey-helldar — CHANGES_REQUESTED** [review]

**Review by andrey-helldar — APPROVED** [review]

**Comment by caouecs** [comment]
merci

**Pull request merged**